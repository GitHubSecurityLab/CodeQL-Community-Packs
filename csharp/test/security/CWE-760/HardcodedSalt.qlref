CWE-760/HardcodedSalt.ql