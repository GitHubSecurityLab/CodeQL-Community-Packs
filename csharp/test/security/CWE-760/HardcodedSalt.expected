edges
| StaticSalt.cs:13:16:13:55 | call to method GetBytes : Byte[] | StaticSalt.cs:14:49:14:52 | access to local variable salt |
| StaticSalt.cs:13:39:13:54 | "Hardcoded Salt" : String | StaticSalt.cs:13:16:13:55 | call to method GetBytes : Byte[] |
| StaticSalt.cs:23:12:23:28 | "Hardcoded Salt2" : String | StaticSalt.cs:29:39:29:63 | call to method generateSalt : String |
| StaticSalt.cs:29:16:29:64 | call to method GetBytes : Byte[] | StaticSalt.cs:30:49:30:52 | access to local variable salt |
| StaticSalt.cs:29:39:29:63 | call to method generateSalt : String | StaticSalt.cs:29:16:29:64 | call to method GetBytes : Byte[] |
nodes
| StaticSalt.cs:13:16:13:55 | call to method GetBytes : Byte[] | semmle.label | call to method GetBytes : Byte[] |
| StaticSalt.cs:13:39:13:54 | "Hardcoded Salt" : String | semmle.label | "Hardcoded Salt" : String |
| StaticSalt.cs:14:49:14:52 | access to local variable salt | semmle.label | access to local variable salt |
| StaticSalt.cs:23:12:23:28 | "Hardcoded Salt2" : String | semmle.label | "Hardcoded Salt2" : String |
| StaticSalt.cs:29:16:29:64 | call to method GetBytes : Byte[] | semmle.label | call to method GetBytes : Byte[] |
| StaticSalt.cs:29:39:29:63 | call to method generateSalt : String | semmle.label | call to method generateSalt : String |
| StaticSalt.cs:30:49:30:52 | access to local variable salt | semmle.label | access to local variable salt |
subpaths
#select
| StaticSalt.cs:14:49:14:52 | access to local variable salt | StaticSalt.cs:13:39:13:54 | "Hardcoded Salt" : String | StaticSalt.cs:14:49:14:52 | access to local variable salt | Use of $@. | StaticSalt.cs:13:39:13:54 | "Hardcoded Salt" | hardcoded salt |
| StaticSalt.cs:30:49:30:52 | access to local variable salt | StaticSalt.cs:23:12:23:28 | "Hardcoded Salt2" : String | StaticSalt.cs:30:49:30:52 | access to local variable salt | Use of $@. | StaticSalt.cs:23:12:23:28 | "Hardcoded Salt2" | hardcoded salt |
